DQM/Integration/python/clients/l1tstage2_dqm_sourceclient-live_cfg.py

```diff
@@ -1,8 +1,8 @@
 import FWCore.ParameterSet.Config as cms
 
 import sys
-from Configuration.Eras.Era_Run3_cff import Run3
-process = cms.Process("L1TStage2DQM", Run3)
+from Configuration.Eras.Era_Run3_2024_cff import Run3_2024
+process = cms.Process("L1TStage2DQM", Run3_2024)
```

Comment:
Just out of curiosity, why is this change needed in this client ? (as opposed to changing only `l1tstage2emulator_dqm`)

Reply:
I think this is not strictly necessary, but if it doesn't change the functionality I would apply the change. It'll be required if we implement something like this to L1T DQM clients